src/helpers/ProgressBar.tsx

```diff
@@ -15,13 +24,14 @@ interface ProgressBarProps {
 const defaultProgressBarProps: Partial<ProgressBarProps> = {
   decimalPoints: 0,
   height: '10px',
+  indicatorThresholds: null,
   max: 100,
   min: 0,
 };
 
 /** Displays configurable progress bar */
 const ProgressBar = (props: ProgressBarProps) => {
```

Comment:
@cKellyDesign there was a considerable amount of work done on the ProgressBar component courtesy of @RayceeM  here https://github.com/onaio/js-tools/pull/95 - I am not sure whether our changes here play nicely with that, and if not how to handle them.  Ideas?

Reply:
@moshthepitt looks like once onaio/js-tools#95 is merged we should install/use it and update the IRS Report Map to use it correctly by passing in `lineColorThresholds` as specified in the package.

Reply:
Issue created at https://github.com/onaio/reveal-frontend/issues/467